### PR TITLE
Initial import of bootloader

### PR DESCRIPTION
**Pull request by AlCutter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,142 @@
...
+|--------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------|--------------------------------------------------------------------------------------|
+| NXP i.MX6UL  | [USB armory Mk II LAN](https://github.com/usbarmory/usbarmory/wiki)                                                                                                                  | [imx6ul](https://github.com/usbarmory/tamago/tree/master/soc/nxp/imx6ul)  | [usbarmory/mk2](https://github.com/usbarmory/tamago/tree/master/board/usbarmory)      |
+
+## Secure Boot
+
+On secure booted systems the `imx_signed` target should be used instead with the relevant
```
Comment (phbnf) [inline README.md:15]: Instead of what?

File: storage/card.go [hunk 2]
```diff
@@ -1,10 +1,16 @@
-// https://github.com/usbarmory/armory-witness-boot
+// Copyright 2022 The ArmoredWitness Authors. All Rights Reserved.
```
Comment (phbnf) [inline storage/card.go:1]: What does this refer to exactly?  The AUTHORS file?

**Review by phbnf — APPROVED** [review]

File: storage/card.go (hunk 2 above)
Comment (AlCutter) [reply storage/card.go:1]: Yes, I can change to Armored Witness Boot authors if you think that's better?

Comment (phbnf) [reply storage/card.go:1]: I will admit that I know very little about copyrights. I don't know if the syntactic sugar used in the .go files, "The ArmoredWitness Authors" needs to be officially defined somewhere? Is an AUTHORS file enough, or does every .go file need to say "The ArmoredWitness Authors are listed in the AUTHORS files of XX"?

Comment (phbnf) [reply storage/card.go:1]: Ah, an AUTHORS file seems to be enough indeed! https://opensource.google/documentation/reference/releasing/authors. Yup, sounds good to me to rephrase this to "Armored Witness Boot" to match with the repo's name indeed.

Comment (AlCutter) [reply storage/card.go]: done
